Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -1492,7 +1492,7 @@ CompletionHandler<void(Decoder*)> Connection::takeAsyncReplyHandler(AsyncReplyID
 bool Connection::isAsyncReplyHandlerWithDispatcher(AsyncReplyID replyID)
 {
     Locker locker { m_incomingMessagesLock };
-    return m_asyncReplyHandlerWithDispatchers.isValidKey(replyID);
+    return m_asyncReplyHandlerWithDispatchers.isValidKey(replyID) && m_asyncReplyHandlerWithDispatchers.contains(replyID);
```

Comment:
Would an invalid key even be in the map? Wondering if the `isValidKey` check is actually necessary.

Reply:
It would't, but it would assert if it was invalid.